From 4524e6eebceb61b3c9835ecd5e6b8f4f4ac6c999 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Tue, 4 Sep 2012 14:21:15 -0400
Subject: [PATCH] OCF/random: update use of mix_pool_bytes() function

commit e0604ba541f04a542ee9a7f6a468a60688ad8c72 in Linux 3.4.9 adds an
out parameter to mix_pool_bytes().  The out is not required.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 drivers/char/random.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/random.c b/drivers/char/random.c
index 7f77760..aa7ead0 100644
--- a/drivers/char/random.c
+++ b/drivers/char/random.c
@@ -821,7 +821,7 @@ void add_disk_randomness(struct gendisk *disk)
  */
 void random_input_words(__u32 *buf, size_t wordcount, int ent_count)
 {
-	mix_pool_bytes(&input_pool, buf, wordcount*4);
+	mix_pool_bytes(&input_pool, buf, wordcount*4, NULL);
 
 	credit_entropy_bits(&input_pool, ent_count);
 
-- 
1.7.9.7

